src/xdocs/config_whitespace.xml

```diff
@@ -600,21 +600,69 @@ class Test {
       </subsection>
 
       <subsection name="Examples" id="FileTabCharacter_Examples">
-        <p>
-          To configure the check to report on the first instance in each
-          file:
-        </p>
+        <p>To configure the check to report only the first instance in each file:</p>
         <source>
 &lt;module name=&quot;FileTabCharacter&quot;/&gt;
         </source>
-        <p>
-          To configure the check to report on each line in each file:
-        </p>
+        <p>Example - Test.java:</p>
+        <source>
+public class Test {
+  int a;                      // violation : indented using tab
+
+  public void foo (int arg) { // OK : indented using tab, only first occurrence in file reported
+      a = arg;                // OK, indented using spaces
+  }                           // OK, indented using spaces
+}
+        </source>
+        <p>To configure the check to report each instance in each file:</p>
         <source>
 &lt;module name=&quot;FileTabCharacter&quot;&gt;
   &lt;property name=&quot;eachLine&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Example - Test.java:</p>
+        <source>
+public class Test {
+  int a;                            // violation : indented using tab
```

Comment:
Remove extra spaces between code and trailing comment

Reply:
Done